public/app/percona/settings/Settings.messages.ts

```diff
@@ -32,6 +32,10 @@ export const Messages = {
     dbaasTooltip:
       'Option to enable/disable DBaaS features. Disabling DBaaS does not suspend or remove running clusters.',
     dbaasLink: `${HOW_TO_DOC_URL}#dbaas`,
+    accessControl: 'Access control',
+    accessControlTooltip: 'Option to enable/disable Access control.',
```

Comment:
can we re-check what's the correct case here?

Reply:
Sorry missed this comment, I'm not sure what you mean here.